src/service/mobile_config.proto

```diff
@@ -81,6 +81,23 @@ message entity_verify_res_v1 {
   bytes signature = 4;
 }
 
+message carrier_key_to_entity_req_v1 {
+  // helium pubkey binary of the carrier
+  string pubkey = 1;
```

Comment:
should be bytes if a pubkeybinary

Reply:
comment updated, intent is to have string representation